sklearn/multioutput.py

```diff
@@ -37,12 +39,90 @@ def _fit_estimator(estimator, X, y, sample_weight=None):
     return estimator
 
 
+def _partial_fit_estimator(estimator, X, y, classes=None, sample_weight=None,
+                           first_time=True):
+    if first_time:
+        estimator = clone(estimator)
+    else:
+        estimator = copy.copy(estimator)
+
+    if sample_weight is not None:
+        if classes is not None:
+            estimator.partial_fit(X, y, classes=classes,
+                                  sample_weight=sample_weight)
+        else:
+            estimator.partial_fit(X, y, sample_weight=sample_weight)
+    else:
+        if classes is not None:
+            estimator.partial_fit(X, y, classes=classes)
+        else:
+            estimator.partial_fit(X, y)
+    return estimator
+
+
 class MultiOutputEstimator(six.with_metaclass(ABCMeta, BaseEstimator)):
 
     def __init__(self, estimator, n_jobs=1):
         self.estimator = estimator
         self.n_jobs = n_jobs
 
+    @if_delegate_has_method('estimator')
+    def partial_fit(self, X, y, classes=None, sample_weight=None):
+        """ Fit linear model with Stochastic Gradient Descent..
+        Fit a separate model for each output variable.
+
+        Parameters
+        ----------
+        X : (sparse) array-like, shape (n_samples, n_features)
+            Data.
+
+        y : (sparse) array-like, shape (n_samples, n_outputs)
+            Multi-output targets. An indicator matrix turns on multilabel
```

Comment:
It doesn't really turn on anything. Multilabel is binary multioutput by definition.